sql/mob_pools.sql

```diff
@@ -985,7 +985,7 @@ INSERT INTO `mob_pools` VALUES (924,'Davoi_Hornet','Davoi_Hornet',48,0x000010010
 INSERT INTO `mob_pools` VALUES (925,'Davoi_Mush','Davoi_Mush',229,0x0000240100000000000000000000000000000000,1,1,7,240,100,0,1,0,0,0,0,0,27,131,0,0,0,1,0,229);
 INSERT INTO `mob_pools` VALUES (926,'Davoi_Pugil','Davoi_Pugil',197,0x00005C0100000000000000000000000000000000,1,1,7,240,100,0,1,0,0,0,0,0,0,3,0,0,0,1,0,197);
 INSERT INTO `mob_pools` VALUES (927,'Davoi_Wasp','Davoi_Wasp',48,0x0000100100000000000000000000000000000000,1,1,2,240,100,0,1,0,1,0,0,0,238,643,0,0,0,1,0,48);
-INSERT INTO `mob_pools` VALUES (928,'Da_Dha_Hundredmask','Da_Dha_Hundredmask',202,0x00008C0200000000000000000000000000000000,6,6,2,265,100,0,1,0,1,2,0,0,0,0,0,0,0,1,0,202);
+INSERT INTO `mob_pools` VALUES (928,'Da_Dha_Hundredmask','Da_Dha_Hundredmask',202,0x00008C0200000000000000000000000000000000,6,6,2,265,100,0,1,0,1,2,2,0,0,0,0,0,0,1,0,202);
```

Comment:
I think Kj prefers we do that anyway. The sql method is in multiple tables, they all says "_mods" in the filename.